dvc/remote/base.py

```diff
@@ -142,6 +138,21 @@ def _check_requires(self, config):
         ).format(url, missing, " ".join(missing), self.scheme)
         raise RemoteMissingDepsError(msg)
 
+    def _get_checksum_jobs(self, repo, config):
```

Comment:
Are we going to have both, _core_ and _remote_ `checksum_jobs`?  Why not going `remote`-only?

Reply:
I've implemented fallback logic as was described in [the comment](https://github.com/iterative/dvc/issues/2920#issuecomment-564128058)

Reply:
@JIoJIaJIu Got it, thanks! Actually, thinking about it, if `checksum_jobs` didn't work anyway, we could consider just getting rid of it, as it doesn't make much sense in many cases. E.g. if core.checksum_jobs is set for 200, then it makes sense for local remote or s3, but will break everything for ssh remote as we will run out of connections. But that will break the backward compatibility, so not worth bothering with right now. So your current solution is correct, let's stick with it for now 👍

Reply:
@efiop , so we are staying with both `core` and `remote` for backward compatibility issues.

Reply:
@mroutis Yep, at least for now.